### ISSUE TITLE
Q: Can the Multiplay Sleep be used on a Realm?

### ISSUE THREAD

**Issue by HeilTec** [Plagiatus/datapacks#10]
I really like the idea of this function of one sleeping for all.

**Comment by Plagiatus** [comment]
Yes. All datapacks that are compatible with the latest (currently 1.15.2) version of the game as well as multiplayer, can be added to a realm.

For this, download the world from your realm, add the datapack to the world and reupload said world.
